packages/@okta/vuepress-site/guides/sign-users-out/sign-out-of-your-app/aspnet/localsignout.md

```diff
@@ -0,0 +1,23 @@
+Open the controller where you will handle the sign out process and add a `SignOut` method to delete the application cookies:
```

Comment:
Open the controller where you handle the sign-out process and add a `SignOut` method to delete the application cookies:

Reply:
I suggest:

> Add a `SignOut` action that accepts an HTTP POST and signs the user out with `CookieAuthenticationDefaults.AuthenticationType`. For example: